src/screens/Data.js

```diff
@@ -0,0 +1,274 @@
+import React from 'react';
+import { Data } from 'grommet';
+import Page from '../components/Page';
+import Item from './Components/Item';
+import {
+  ComponentDoc,
+  Properties,
+  Property,
+  PropertyValue,
+  Description,
+  Example,
+} from '../components/Doc';
+
+const propertiesObjectExample = `{
+  name: {
+    label: 'Name',
+    search: true,
+  },
+  country: {
+    label: 'Country',
+    options: ['US', 'UK', 'FR'],
+  },
+  age: {
+    label: 'Age',
+    range: {
+      min: 1,
+      max: 120,
+    },
+  },
+}`;
+
+const viewObjectExample = `{
+  properties: {
+    country: ['US', 'UK'],
+    age: {
+      min: 22,
+      max: 56,
+    },
+  },
+  search: 'com',
+  sort: {
+    property: 'date',
+    direction: 'desc',
+  },
+  page: 1,
+  step: 50
+}`;
+
+const viewsObjectExample = `[
+  {
+    name: 'English speaking',
+    properties: {
+      country: ['US', 'UK'],
+    },
+    search: 'comm',
+    sort: {
+      property: 'date',
+      direction: 'desc',
+    },
+    step: 50
+  },
+  {
+    name: 'middle age',
+    properties: {
+      age: {
+        min: 40,
+        max: 60,
+      },
+    },
+    sort: {
+      property: 'age',
+      direction: 'asc',
+    },
+    step: 50
+  },
+]`;
+
+const messagesObjectExample = `{
+  "dataFilters": {
+    "clear": "Clear filters",
+    "heading": "Filters",
+    "open": "Open filters"
+  },
+  "dataForm": {
+    "reset": "Undo changes",
+    "submit": "Apply filters"
+  },
+  "dataSearch": {
+    "label": "Search data"
+  },
+  "dataSort": {
+    "ascending": "Ascending",
+    "by": "Sort by",
+    "descending": "Descending",
+    "direction": "Sort direction"
+  },
+  "dataSummary": {
+    "filtered": "{filteredTotal} results of {total} items",
+    "filteredSingle": "{filteredTotal} result of {total} items",
+    "total": "{total} items"
+  },
+  "dataTableColumns": {
+    "open": "Open column selector",
+    "order": "Order columns",
+    "select": "Select columns"
+  },
+}`;
+
+const DataPage = () => (
+  <Page>
+    <ComponentDoc
+      name="Data"
+      stable
+      availableAt={[
+        {
+          url: 'https://storybook.grommet.io/?selectedKind=Layout-Footer&full=0&stories=1&panelRight=0',
+          badge:
+            'https://cdn-images-1.medium.com/fit/c/120/120/1*TD1P0HtIH9zF0UEH28zYtw.png',
+          label: 'Storybook',
+        },
+        {
+          url: 'https://github.com/grommet/grommet/tree/master/src/js/components/Data',
+          label: 'Github',
+        },
+      ]}
+      description="Container and context for data driven content"
```

Comment:
```suggestion
      description="Container and context provider for data driven content"
```
is this too technical?